docs/tutorials/resource-migration.md

```diff
@@ -0,0 +1,295 @@
+* `spec.resourceSelectors`：defining which resources are selected to migrate, you can define your custom [ResourceSelector](https://karmada.io/docs/userguide/scheduling/override-policy/#resource-selector).
+
+```shell
+➜ ✗ kubectl --context karmada-apiserver apply -f /tmp/migrate-pp.yaml
+propagationpolicy.policy.karmada.io/migrate-pp created
+```
+
+Now, you have finished the migration, isn't it so easy? You can do the verification as:
+
+```shell
+➜ ✗ kubectl --context karmada-apiserver get deploy
+➜ ✗ kubectl --context karmada-apiserver get rb
+```
+
+You can see the Deployments in Karmada are all ready and the `aggregatedStatus` of `ResourceBinding` is applied.
+
+### Tutorial 2: clone application from member1 to member2 cluster
+
+Apply [/tmp/nginx-pp.yaml](#nginx-ppyaml) to Karmada control plane. 
+
+```shell
+➜ ✗ kubectl --context karmada-apiserver apply -f /tmp/nginx-pp.yaml
+propagationpolicy.policy.karmada.io/nginx-pp created
+```
+
+Then you will find `nginx` application related resource are all copied to `member2` cluster:
+
+```shell
+➜ ✗ kubectl --context member2 get deploy -o wide                                                 
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE     CONTAINERS   IMAGES         SELECTOR
+nginx-deploy   2/2     2            2           5m24s   nginx        nginx:latest   app=nginx
+➜ ✗ kubectl --context member2 get svc -o wide   
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE   SELECTOR
+nginx-svc    NodePort    10.13.161.255   <none>        80:30000/TCP   54s   app=nginx
+...
+```
+
+### Tutorial 3: expand application to member2 with sum replicas number unchanged
+
+Apply [/tmp/hello-pp.yaml](#hello-ppyaml) to Karmada control plane.
+
+```yaml
+➜ ✗ kubectl --context karmada-apiserver apply -f /tmp/hello-pp.yaml
+propagationpolicy.policy.karmada.io/hello-pp created
+```
+
+Then you will find `hello` application related resource are expand to `member2` cluster:
+
+```shell
+➜ ✗ kubectl --context member1 get deploy -o wide
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE     CONTAINERS   IMAGES         SELECTOR
+hello-deploy   1/1     1            1           5m51s   nginx        nginx:latest   app=hello
+...
+➜ ✗ kubectl --context member2 get deploy -o wide
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE     CONTAINERS   IMAGES         SELECTOR
+hello-deploy   1/1     1            1           2m28s   nginx        nginx:latest   app=hello
+...
+➜ ✗ kubectl --context member2 get svc -o wide   
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE     SELECTOR
+hello-svc    NodePort    10.13.210.81    <none>        8080:30080/TCP   2m51s   app=hello
+...
+```
+
+You can see the total replicas number of `hello-deploy` in all clusters sum to 2, which is as defined in the raw Deployment.
```

Comment:
This is out of the scope of this tutorial. we can remove it.

Reply:
hello, docs has been improved, [quick click](https://deploy-preview-431--karmada.netlify.app/docs/next/tutorials/resource-migration/)